portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -1,15 +1,16 @@
 package com.google.sps.servlets;
 public class Comment {
 
-  //When user authenticaton is implemented, this will change.
-  String user = "Anonymous";
+    //When user authenticaton is implemented, this will change.
+    String user = "Anonymous";
```

Comment:
Theoretically this can just be deleted and added when you need it, since it's unused right now.